R/latent_individual.R

```diff
@@ -70,6 +64,31 @@ epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
   )
 }
 
+#' Define priors for the model
+#' 
+#' We provide suggested priors for the intercepts of the `meanlog` and `sdlog`
+#' parameters of the delay distribution. These priors are weakly informative in
+#' that they prevent excessive delays on the real scale. In particular:
+#'
+#' * `meanlog ~ normal(2, 0.5)`
+#' * `sdlog ~ normal(0.5, 0.5)`
+#'
+#' To alter the priors for this model, we recommend first fitting using these
+#' default priors, then extracting the priors with `brms::get_prior()`.
+#' Elements of this `data.frame` of priors may then be modified, and the
+#' resulting object passed to `epidist` to be used. An example of this workflow
+#' will be provided as a part of a vignette at a future date.
+#'
+#' @inheritParams epidist_prior
+#' @method epidist_prior epidist_latent_individual
+#' @family latent_individual
+#' @export
+epidist_prior.epidist_latent_individual <- function(data, ...) {
+  prior1 <- brms::prior("normal(2, 0.5)", class = "Intercept")
+  prior2 <- brms::prior("normal(0.5, 0.5)", class = "Intercept", dpar = "sigma")
```

Comment:
if this is one the log scale is it a reasonable prior?

Reply:
I'm not sure what you mean here?

Reply:
Do you have an idea about what mean on the real scale you'd like? I just picked some reasonable seeming numbers but I'm open to changing (I didn't really use my expertise about delays because I don't have much). Of course we should be having something pretty wide.

A more sophisticated version I'd like what 5% and 95% numbers then back calculate to get the two priors.

Reply:
my point is that I don't think this is a prior on sdlog  but on sdlog_log.

Reply:
Yes that sounds right to me and I'll change things to fit with it being on `sdlog_log`